### PR TITLE
Do not require gradle.properties for build

### PR DESCRIPTION
**Pull request by losipiuk**
Do not require gradle.properties for build

reviewers: gkokosinski,ksobczak

### CODE REVIEW

**Comment by sogorkis** [comment]
lgtm

File: build.gradle [hunk 1]
```diff
@@ -98,6 +98,10 @@ ext.tempto_version = '1.0.36-SNAPSHOT' // changed by losipiuk
 ext.tempto_group = "com.teradata.tempto"
 ext.isReleaseVersion = !tempto_version.endsWith("SNAPSHOT")
 
+ext.nexusUsername = 'setup_nexusUsername_in_gradle.properties'
+ext.nexusPassword = 'setup_nexusPassword_in_gradle.properties'
+ext.skipSigning = false
```
Comment (kokosing) [inline build.gradle]: true?

File: gradle.properties.example [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
-signing.secretKeyRingFile=KEY_PATH
+#skipSigning=false
+#signing.keyId=KEYID
+#signing.password=KEY_PASSWORD
+#signing.secretKeyRingFile=KEY_PATH
 
-nexusUsername=SONATYPE_LOGIN
-nexusPassword=SONATYPE_PASSWORD
+#nexusUsername=SONATYPE_LOGIN
+#nexusPassword=SONATYPE_PASSWORD
```
Comment (kokosing) [inline gradle.properties.example]: why it is commented?

**Comment by kokosing** [comment]
lgtm % comments

File: gradle.properties.example (hunk 2 above)
Comment (losipiuk) [inline gradle.properties.example]: For some reason if you just copy build.properties.example to build.properties and leave broken entries for sign* properties the build will fail.

It works if sign* properties are not defined at all.

**Pull request merged**